homeassistant/components/homewizard/quality_scale.yaml

```diff
@@ -0,0 +1,93 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      The integration does not provide any additional actions.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow:
+    status: todo
+    comment: |
+      The data_descriptions are missing.
+  dependency-transparency: done
+  docs-actions:
+    status: todo
+    comment: |
+      The integration does not provide any additional actions.
```

Comment:
Should this one be set to 'exempt' or do we need to provide docs that there are no actions?